README.md

```diff
@@ -182,6 +182,61 @@ tell it to do.
 
 Enjoy streaming Google Music via Alexa!
 
+### (Optional) Setting up a Heroku instance
+
+Setting up an instance on Heroku may be an easier option for you, and these
+instructions detail how to accomplish this. The following steps replace the need
+to setup a local server. First one must have Heroku setup on your local machine and
+an account associated. Go to https://heroku.com for information about this.
+
+One must then clone the repository.
+
+```bash
+$ git clone https://github.com/stevenleeg/geemusic.git
+```
+
+Next, `cd` in to quickly modify and then deploy the code.
+
+Now, open server.py in your text editor, and at the top add the following line of
+code to the top.
+
+```
+import os
+```
+
+Then, change `app.run(debug=True)` to
+
+```
+app.run(host="0.0.0.0", port=int(os.environ["PORT"]), debug=True)
```

Comment:
Rather than instructing people to modify files/check them into git, I think a better approach would be to change `server.py` in this PR to use `os.environ['PORT']` but provide a sane default, `4000` if the key doesn't exist in the environment.